ci/checks/style.sh

```diff
@@ -38,16 +38,31 @@ source ci/common/prep-init-env.sh ${WORKSPACE}
 logger "Run Copyright header check..."
 ./ci/checks/check_copyright.py
 
+# Python
 logger "Run Python/Cython formatting check..."
 python -m pip install -r ./ci/checks/python-style-requirements.txt
 source pyclaragenomics/style_check
 
+logger "Run Python documentation generation..."
+mkdir -p pyclaragenomics/docs; cd pyclaragenomics/docs
+sphinx-quickstart -p "Clara Genomics Analysis SDK" -v "x.y.z" -a "NVIDIA Corportation" -q --ext-autodoc --sep
```

Comment:
We should extract the version from setup.py for now (until we create a setup.cfg file).

Reply:
I specified a dummy version in this line mainly because this doesn't create the official documentation. It's just a test to ensure that documentation generation runs successfully. Working on a separate release script that will do proper version reading and so on.